README.md

```diff
@@ -188,8 +206,17 @@ This strategy makes sure that pods violating interpod anti-affinity are removed
 if there is podA on a node and podB and podC (running on the same node) have anti-affinity rules which prohibit
 them to run on the same node, then podA will be evicted from the node so that podB and podC could run. This
 issue could happen, when the anti-affinity rules for podB and podC are created when they are already running on
-node. Currently, there are no parameters associated with this strategy. To disable this strategy, the
-policy should look like:
+node. Currently, there are no parameters associated with this strategy.
```

Comment:
`Currently, there are no parameters associated with this strategy.` not necessary here

Reply:
`To disable this strategy, the policy should look like` still applies here unless the example's `enabled` field is set to `true`

Reply:
updated